grouping.js

```diff
@@ -3,7 +3,7 @@ let allowDirectIdSelectors = false;
 
 // Publish admin and group for users that have it
 Meteor.publish(null, function () {
-	return this.userId && Meteor.users._partitionerDirect.find(this.userId, {fields: {admin: 1, group: 1}});
+	return this.userId && Meteor.users._partitionerDirect.find(this.userId, {projection: {admin: 1, group: 1}});
```

Comment:
```suggestion
// Since Meteor.release@2.16 {projection: {}} has been preferred over the deprecated {fields: {}}
// Note that string comparison breaks since "2.16" is newer than "2.9" - so need to pad with zeros
let release = Meteor.release.split('@')[1].split('.')
release = release[0] + '.' + release[1].padStart(2, '0')
let defaultProjectionField = parseFloat(release) >= 2.16 ? 'projection' : 'fields';

// Publish admin and group for users that have it
Meteor.publish(null, function () {
	return this.userId && Meteor.users._partitionerDirect.find(this.userId, {[defaultProjectionField]: {admin: 1, group: 1}});
```

Reply:
What is the a reason you check for 2.16 instead of 2.6 here?

Reply:
Sorry, my mistake.  I was testing this on a project on 2.16, so I had that number in my head when I wrote this.  The test, when converted to floating point, should be `parseFloat(release) >= 2.06`.

Well spotted, thanks.